### ISSUE TITLE
Regarding overfitting

### ISSUE THREAD

**Issue by oasis-0927** [Tony607/Yelp_review_generation#2]
Hi. I've been trying to generate some Chinese reviews. One thing I noticed is that in your implementation and several similar tasks is that there is no validation process to quantify the val_loss and accuracy. And therefore I can't tell whether the model is overfitting.  I added those metrics in several projects and I think they overfitted. Is this designed this way? Like, it was supposed to be 'overfitted' to the training data because the training data is already comprehensive and well-coverd?

Thank you for your time.
